packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -886,7 +884,7 @@ class _SearchField extends StatelessWidget {
 
 class SearchDropdown<T> extends StatelessWidget {
```

Comment:
looks like this class is not used anywhere and can be removed

Reply:
done